src/convertToCase/inputValidation.js

```diff
@@ -0,0 +1,33 @@
+const allCases = ['SNAKE', 'KEBAB', 'UPPER', 'CAMEL', 'PASCAL'];
+
+function inputValidation(originalText, targetCase) {
+  const errors = [];
+
+  if (!originalText) {
+    errors.push({
+      message:
+        // eslint-disable-next-line max-len
+        'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
```

Comment:
It seems like you've accidentally left a blank line at line 9. It's better to remove it to keep your code clean and readable.